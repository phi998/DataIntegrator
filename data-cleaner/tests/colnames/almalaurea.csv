Company Name,Salary,Advertisement Title,Company Name,Tipo candidato,Publication Date,Industry,Location,Advertisement Content,Advertisement Content,Advertisement Content,Advertisement Title,Contract type,Industry,Advertisement Content,Advertisement Content,Publication Date
Company Name,Salary,Advertisement Title,Company Name,Contract Type,Publication Date,Industry,Location,Advertisement Content,Advertisement Content,Advertisement Content,Advertisement Content,Contract type,Industry,Advertisement Content,Company Name,Publication Date
