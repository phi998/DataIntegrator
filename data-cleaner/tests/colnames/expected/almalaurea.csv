Company Name,Salary,Advertisement Title,Company Name,Tipo candidato,Publication Date,Industry,Location,Advertisement Content,Advertisement Content,Advertisement Content,Advertisement Title,Contract type,Industry,Industry,Advertisement Content,Publication Date
